workflows/templates/advise-template.yaml

```diff
@@ -43,7 +43,7 @@ spec:
             archive:
               none: {}
             s3:
-              key: "argo/adviser/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+              key: "data/thoth/thoth-psi-stage/adviser/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
```

Comment:
Can this be obtained from `thoth` configmap not to hardcode it to the template?

Reply:
there is way, @CermakM suggested me one solution, I will implement it in another PR :)

Reply:
So I guess you want this one to be merged, or will it replace this one? :)

Reply:
This can be merged meanwhile :) I need to test the new solution.

Reply:
:crossed_fingers: